pyperformance/data-files/benchmarks/bm_yaml/run_benchmark.py

```diff
@@ -0,0 +1,91 @@
+"""
+Script for testing the performance of YAML parsing, using yaml.
+
+This will dump/load several real world-representative objects a few thousand
+times. The methodology below was chosen for was chosen to be similar to
+real-world scenarios which operate on single objects at a time.
+
+This explicitly tests the pure Python implementation in pyyaml, not its C
+extension.
+
+The object structure is copied from the `json_load` benchmark.
+"""
+
+
+import random
+import sys
+
+
+import pyperf
+import yaml
+
+
+DICT = {
+    'ads_flags': 0,
+    'age': 18,
+    'bulletin_count': 0,
+    'comment_count': 0,
+    'country': 'BR',
+    'encrypted_id': 'G9urXXAJwjE',
+    'favorite_count': 9,
+    'first_name': '',
+    'flags': 412317970704,
+    'friend_count': 0,
+    'gender': 'm',
+    'gender_for_display': 'Male',
+    'id': 302935349,
+    'is_custom_profile_icon': 0,
+    'last_name': '',
+    'locale_preference': 'pt_BR',
+    'member': 0,
+    'tags': ['a', 'b', 'c', 'd', 'e', 'f', 'g'],
+    'profile_foo_id': 827119638,
+    'secure_encrypted_id': 'Z_xxx2dYx3t4YAdnmfgyKw',
+    'session_number': 2,
+    'signup_id': '201-19225-223',
+    'status': 'A',
+    'theme': 1,
+    'time_created': 1225237014,
+    'time_updated': 1233134493,
+    'unread_message_count': 0,
+    'user_group': '0',
+    'username': 'collinwinter',
+    'play_count': 9,
+    'view_count': 7,
+    'zip': ''}
+
+TUPLE = (
+    [265867233, 265868503, 265252341, 265243910, 265879514,
+     266219766, 266021701, 265843726, 265592821, 265246784,
+     265853180, 45526486, 265463699, 265848143, 265863062,
+     265392591, 265877490, 265823665, 265828884, 265753032], 60)
```

Comment:
It may be worth it to also/instead use some real-world data, like we do for bm_tomli_loads?